tests-user/tests/create-lz-oapp.bats

```diff
@@ -264,6 +286,17 @@ teardown() {
     npm run lint:fix
 }
 
+@test "should work with npm & native-oft-adapter example in CI mode" {
+    local DESTINATION="$PROJECTS_DIRECTORY/npm-oft"
```

Comment:
Same question as above here -- Should this be `npm-native-oft-adapter` instead? I copied this over from the oft-adapter test right above which also lists the destination as `npm-oft`. I can fix both if this is a typo